apps/ui/src/Communities/CommunitiesFeed.tsx

```diff
@@ -22,6 +24,8 @@ const CommunitiesFeed: React.FC<CommunitiesFeedProps> = ({ communities }) => {
   const userCommunitiesStatusOriginal = useUserCommunitiesStatus(
     user?.id ?? 'admin',
   );
+
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
```

Comment:
why have this in each of the page components and not have this in one component that apper everywhere ?